server/routes/charts.js

```diff
@@ -165,8 +178,14 @@ router
     try {
       const { dataDb } = req.app.locals
       const { chart_id } = req.params
-      const { title, variable, assessment, description, fieldLabelValueMap } =
-        req.body
+      const {
+        title,
+        variable,
+        assessment,
+        description,
+        fieldLabelValueMap,
+        public: isPublic,
```

Comment:
would it make sense to call this field as `isPublic` everywhere?

Reply:
In the issue I think we want it to be public property. Yeah the workaround is different.
https://github.com/AMP-SCZ/dpdash/issues/140

Reply:
😕 I'm not sure what you mean, are you saying the requirement from the client is that it be called `public`?

Reply:
yes the requirement is for it to be called public

Reply:
well, that settles that then 😄